### PR TITLE
implement footer with data

### PR DESCRIPTION
**Pull request by rjspiker**

### CODE REVIEW

File: components/footer/index.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+class Footer extends React.Component {
+  render() {
+    const footer = this.props.data.globalFooter
+
+    return (
+      <footer className="g-footer">
+        <div className="g-container">
+          <div className="top">
+            {/* <NewsletterSignupForm placement="footer" /> */}
```
Comment (rjspiker) [inline components/footer/index.js]: can deal with this later

File: components/footer/index.js [hunk 2]
```diff
@@ -0,0 +1,161 @@
+import React from 'react'
+import { graphql } from 'react-apollo'
+import getFooter from './getFooter.graphql'
+
+import styles from './style.css'
+
+class Footer extends React.Component {
```
[the remark this answers is not in the document]
Comment (rjspiker) [reply components/footer/index.js]: Ya.  I just wasn't sure which convention we were going with.  I can make that change.

**Comment by rjspiker** [comment]
OK, made a couple of adjustments based on @mwickett 's feedback.

**Comment by schonfeld** [comment]
migrated over to master. no need for this.

**Pull request closed without merging**